db/db_impl/db_impl_compaction_flush.cc

```diff
@@ -1649,6 +1649,10 @@ Status DBImpl::ReFitLevel(ColumnFamilyData* cfd, int level, int target_level) {
 
   InstrumentedMutexLock guard_lock(&mutex_);
 
+  auto* vstorage = cfd->current()->storage_info();
```

Comment:
[Note] Decided to early return to avoid `cfd->compaction_picker()->GetRange()` below on empty input.